common/sgx/workload/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ SET(GENERIC_CXX_FLAGS ${DEBUG_FLAGS} "-Wall" "-fPIC" "-Wno-write-strings" "-std=
 
 INCLUDE_DIRECTORIES(${TCF_TOP_DIR}/tcs/core/tcs_trusted_worker_manager/enclave)
 
-SET(WORKLOAD_STATIC_NAME sgx_workload)
+SET(WORKLOAD_STATIC_NAME workload)
```

Comment:
Could you rename the static library as common_sgx_workload. workload -> common_sgx_workload

Reply:
Done